network-api/networkapi/buyersguide/pagemodels/product_category.py

```diff
@@ -43,7 +43,7 @@ def __str__(self):
         return self.name
 
     def save(self, *args, **kwargs):
-        self.slug = slugify(self.name)
+        self.slug = slugify(self.name_en)
```

Comment:
do we want fallback? e,g, `self.name_en if self.name_en else self.name`?

Reply:
Yes it wouldnt hurt to have this imo. @TheoChevalier Would know more about this, but I'm in favour of having a fallback.

Reply:
Having a fallback sounds good to me, I supposed enabling localization meant we always had a `name_en` value, like we always had `name`